python/seldon_core/proto/tensorflow/core/framework/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 if _TF_PRESENT:
     # Let tensorflow shadow these imports if present
-    from tensorflow.core.framework import tensor_pb2, types_pb2, tensor_shape_pb2
+    from tensorflow.core.framework import tensor_pb2, tensor_shape_pb2, types_pb2
```

Comment:
We want to exclude protos also from `isort`.

Reply:
This isn't about formatting pb2 files per se, it just formats how they're formatted

Reply:
Hmm. But this is inside the proto for, let me see

Reply:
> This isn't about formatting pb2 files per se, it just formats how they're formatted

Now that's a definitely a bigger change that should go into separate issue for discussion ; - )

Reply:
Sorry I meant how they're **imported***.

Ok, I agree this issue is getting overly large, I'll limit the change to seldon_core only.

Reply:
I think the principal is that every manually written python files (except for auto generated ones e.g pb2 files) should follow consistent formatting enforced by black and isort, is that agreeable?

Reply:
Yes, that's right. What we write manually should be under `black` and `isort`. So all python codes except those under `proto` should fall into it. 

The tensorflow protos I think I coming from 3rd party library so we should not format them.

Reply:
I see. I probably committed this one by mistake as `proto` directory has been indeed ignored. I've changed it back.